CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ endif()
 # Point CMake at any custom modules we may ship
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  set(CMAKE_SYSTEM_PROCESSOR arm64)
```

Comment:
According to https://stackoverflow.com/a/70498851/152142 , this is probably fine. Though it mentions ARMClang args may be affected, and in theory, `Modules/CMake<LANG>Information.cmake` files are selected based on this env var.

To mitigate "risk", does it work if we only set this in `cmake.packaging/CMakeLists.txt` or somehow restrict it to the "packaging step" (cpack)?

Reply:
Those files are selected before though, in the `project` function call. But we can set it in packaging as well and it should work, so let's do that.